.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "flatpak-builder-tools"]
+	path = flatpak-builder-tools
+	url = git@github.com:flatpak/flatpak-builder-tools.git
```

Comment:
this should be the https URL. (BOTH)

Reply:
Ok. May I ask why? 
I've had issues with this in the past when the main repo is ssh/git but the submodules are https. Especially since github no longer allows git authorization over https... I'll change it ofc, but would just like to understand the reasoning.

Reply:
they don't need permission and using git require setup ssh keys, which, of course, the builders don't have.

Reply:
:+1:  ok, thanks. Thought that was probably why.